Src/Support/Google.Apis.Auth/OAuth2/GoogleClientSecrets.cs

```diff
@@ -50,9 +50,25 @@ public ClientSecrets Secrets
         }
 
         /// <summary>Loads the Google client secret from the input stream.</summary>
-        public static GoogleClientSecrets Load(Stream stream)
+        public static GoogleClientSecrets Load(Stream stream) =>
+            NewtonsoftJsonSerializer.Instance.Deserialize<GoogleClientSecrets>(stream);
+
+        /// <summary>Loads the Google client secret from the input stream.</summary>
+        public static Task<GoogleClientSecrets> LoadAsync(Stream stream, CancellationToken cancellationToken = default) =>
+            NewtonsoftJsonSerializer.Instance.DeserializeAsync<GoogleClientSecrets>(stream, cancellationToken);
+
+        /// <summary> Loads the Google client secret from a JSON file.</summary>
+        public static GoogleClientSecrets Load(string clientSecretsFilePath)
```

Comment:
How about calling these FromStream, FromStreamAsync, FromFile and FromFileAsync? That way:

- We can add FromJson later if we want to
- It's consistent with the GoogleCredential factory methods

Reply:
Yep, that sounds good. But, we'd need to keep the original `Load` around so as not to break anyone. Maybe we can even mark it as deprecated since we re offering an exact alternative, save for the name. What do you think?

Reply:
Ah, I hadn't noticed that already existed. Yes, I think deprecating it makes sense.

Reply:
Done, PTAL! Thanks